modules/logger/parser/integration_test.go

```diff
@@ -0,0 +1,66 @@
+package parser
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/shell"
+	"github.com/stretchr/testify/assert"
+)
+
+func DirectoryEqual(t *testing.T, dirA string, dirB string) bool {
+	dirAAbs, err := filepath.Abs(dirA)
+	if err != nil {
+		t.Fatal(err)
+	}
+	dirBAbs, err := filepath.Abs(dirB)
+	if err != nil {
+		t.Fatal(err)
+	}
+	cmd := shell.Command{
+		Command: "diff",
+		Args:    []string{"-arq", dirAAbs, dirBAbs},
```

Comment:
Add comment for why we're using `diff` (as opposed to something written in Go).

Reply:
fixed